imageio/core/util.py

```diff
@@ -568,7 +568,7 @@ def has_module(module_name):
         import imp
 
         try:
-            imp.find_module(module_name)
+            imp.find_module(module_name, package)
```

Comment:
The second arg of find_module must be a path. Not sure how to do paths with this approach tbh, so this is kinda broken on 27.
```suggestion
        imp.find_module(module_name)
```